client/agenda/AgendaMobileBar.vue

```diff
@@ -31,7 +31,7 @@ import {
 } from 'naive-ui'
 
 import { useAgendaStore } from './store'
-import { useSiteStore } from '../shared/store';
+import { useSiteStore } from '../shared/store'
```

Comment:
Superfluous change?

Reply:
There are many files in here that shouldn't be - I suspect Eric was based in the wrong place. I'll work with him to pull out the parts of this that were meant to change.

Reply:
Indeed, I tried to start from feat/tzaware rather than main but I failed.

Reply:
Will redo the changes in a fresh PR